modules/querier/worker/frontend_processor.go

```diff
@@ -27,6 +29,12 @@ var processorBackoffConfig = backoff.Config{
 	MaxBackoff: 1 * time.Second,
 }
 
+var metricWorkerRequests = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
i don't mind this but do you expect it to be significantly different than the querier request rates?

Reply:
yup, I think request rate is for batches, and this is at job level (single request). also, this is counter so we can capture how much work each querier is doing...